src/server/main_service.cc

```diff
@@ -1465,6 +1485,9 @@ void Service::Quit(CmdArgList args, ConnectionContext* cntx) {
     cntx->SendOk();
   using facade::SinkReplyBuilder;
 
+  // turn off tracking for this client
```

Comment:
No need for it here,.this path calls OnClose

Reply:
fixed.